docs/development/UX/strategy.md

```diff
@@ -0,0 +1,48 @@
+# UX strategy
+
```

Comment:
I think we are missing the fact that we have 3 devices: HW - K - SMF
And it is quite clear that they cannot be far from each other.
It follows that the conditional UX user cannot be anywhere, but only where it is convenient to perform the necessary procedures and only where there is a stable connection. Then we can imagine a certain conditional scheme of the functioning of the overall system:

Reply:
See the terminology in suggestion. Let's not label anything but Hot Wallet with HW, or get rid of these abbreviation altogether.